daemon/api.go

```diff
@@ -585,21 +585,52 @@ func deleteOp(c *Command, r *http.Request) Response {
 	}
 }
 
+type licenseData struct {
```

Comment:
Would you mind to drop the internal comment and add a type comment above for clarity. Something along these lines:
```
// licenseData holds details about the snap license, and may be marshaled 
// back as an error when the license agreement is pending. As such, its
// field names are part of the API.
```